demo-setups/fetchmail-compose.yaml

```diff
@@ -1,10 +1,10 @@
-# Docs: https://docker-mailserver.github.io/docker-mailserver/v14.0/config/advanced/mail-fetchmail
+# Docs: https://docker-mailserver.github.io/docker-mailserver/latest/config/advanced/mail-fetchmail
```

Comment:
Revert this, the docs page is going to get relocated in another release, thus the link will break in future. An interested user will be able to infer the version in the URL if they want to view newer release docs.

The intent along with the comment below was to indicate that the last known compatible configuration for the example was with that DMS release (_which is what was used for the example it links to for CLI commands_). We have no tests for these, so the version info minimizes maintenance or troubleshooting should users report an issue 😅

Reply:
I see you already applied this. I'm resolving this comment.